src/SessionMonitor.ts

```diff
@@ -17,22 +17,17 @@ export class SessionMonitor {
     private _checkSessionIFrame?: CheckSessionIFrame;
 
     public constructor(private readonly _userManager: UserManager) {
-        if (!_userManager) {
```

Comment:
no logger refactor, should be in a separate commit

Reply:
I also didn't realize that we export SessionMonitor from the entrypoint so this check should stay anyway